sdks/java/io/hadoop-format/build.gradle

```diff
@@ -140,4 +157,12 @@ task hadoopFormatIOElasticTest(type: Test) {
   testClassesDirs = sourceSets.test.output.classesDirs
   include '**/HadoopFormatIOElasticIT.class'
   useJUnit { }
-}
\ No newline at end of file
+}
+
+task hadoopVersion285Test(type: Test) {
+  dependsOn createTargetDirectoryForCassandra
+  description = "Runs Hadoop format tests with Hadoop version $hadoop_285_version"
+  outputs.upToDateWhen { false }
+  include '**/*Test.class'
+  include '**/HadoopFormatIOElasticIT.class'
```

Comment:
Without looking in detail, this looks like it might be worth checking. Seems to set up for Cassandra, but runs ES tests?

Reply:
Oh right, it makes no sense to run `HadoopFormatIOElasticIT` here. This test has a default configuration that can be run without setting up anything but it indeed is inconsistent with the whole idea of `hadoopVersion285Test`

Reply:
Cassandra is set for `HadoopFormatIOCassandraTest`